chain/network/src/types.rs

```diff
@@ -795,7 +900,7 @@ pub struct NetworkInfo {
     pub sent_bytes_per_sec: u64,
     pub received_bytes_per_sec: u64,
     // Only send full routes to accounts on demand
-    pub routes: Option<HashMap<AccountId, (PeerId, usize)>>,
+    pub routes: Option<RoutingTable>,
```

Comment:
I'm going to change this part a bit - we don't need the routing table, we just need to know which block / chunk producers we have connected to but we need to send it always - because if we don't have enough connections as block producer should pause and wait to connect / discover more.

Reply:
This here was only for tests. When we request network info I added a **verbose** level so we onle receive Routing Table with level >= 1.